src/test/resources/crashes/minecraft/true-IllegalArgumentException-level-file.txt

```diff
@@ -0,0 +1,187 @@
+	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
+	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
+	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
+	at cjb.a(SourceFile:22)
+	at cjb.a(SourceFile:14)
+	at civ.a(SourceFile:55)
+
+-- Feature --
+Details:
+	Id: minecraft:decorated
+	Config: < clv [minecraft:decorated | [minecraft:count_noise_biased cql@6e90e03b]] >
+	Description: < cjb [minecraft:decorated] >
+Stacktrace:
+	at bsv.a(SourceFile:373)
+
+-- Generation --
+Details:
+	CenterX: -47
+	CenterZ: 6192
+	Seed: 2734632675027547379
+	Biome: bsv@37736bf1
+Stacktrace:
+	at cfy.a(SourceFile:220)
+	at cga.c(SourceFile:89)
+	at cga.a(SourceFile:219)
+
+-- Chunk to be generated --
+Details:
+	Location: -47,6192
+	Position hash: 26598732464081
+	Generator: cho@382fd5f3
+Stacktrace:
+	at zs.a(SourceFile:568)
+	at com.mojang.datafixers.util.Either$Left.map(Either.java:38)
+	at zs.a(SourceFile:562)
+	at java.base/java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1072)
+	at java.base/java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:478)
+	at zu.b(SourceFile:58)
+	at aoe.e(SourceFile:94)
+
+-- Entity being ticked --
+Details:
+	Entity Type: minecraft:falling_block (bcu)
+	Entity ID: 2126
+	Entity Name: Falling Block
+	Entity's Exact location: -702.50, 30.01, 99062.50
+	Entity's Block location: World: (-703,30,99062), Chunk: (at 1,1,6 in -44,6191; contains blocks -704,0,99056 to -689,255,99071), Region: (-2,193; contains chunks -64,6176 to -33,6207, blocks -1024,0,98816 to -513,255,99327)
+	Entity's Momentum: 0.00, 0.00, 0.00
+	Entity's Passengers: []
+	Entity's Vehicle: ~~ERROR~~ NullPointerException: null
+	Immitating BlockState: Block{minecraft:sand}
+Stacktrace:
+	at brx.a(SourceFile:564)
+	at aag.a(SourceFile:411)
+	at net.minecraft.server.MinecraftServer.b(SourceFile:871)
+	at zg.b(SourceFile:312)
+	at net.minecraft.server.MinecraftServer.a(SourceFile:811)
+	at net.minecraft.server.MinecraftServer.w(SourceFile:670)
+	at net.minecraft.server.MinecraftServer.a(SourceFile:257)
+	at java.base/java.lang.Thread.run(Thread.java:834)
+
+-- Affected level --
+Details:
+	All players: 1 total; [aah['Marcono1234'/292, l='ServerLevel[world]', x=-659.01, y=95.90, z=99029.02]]
+	Chunk stats: ServerChunkCache: 3079
+	Level dimension: minecraft:overworld
+	Level spawn location: World: (-208,67,208), Chunk: (at 0,4,0 in -13,13; contains blocks -208,0,208 to -193,255,223), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)
+	Level time: 17947 game time, 17947 day time
+	Level name: world
+	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
+	Level weather: Rain time: 140516 (now: false), thunder time: 23783 (now: false)
+	Known server brands: vanilla
+	Level was modded: true
```

Comment:
Note that I manually edited this value. If we want a more authentic crash report we might be able to use one from Mojira.